src/CopyrighFixer/CopyrighFixer/copyrighfixer.h

```diff
@@ -14,4 +15,25 @@ namespace CopyrighFixer {
 
 bool CopyrighFixer_EXPORT init();
 
+/**
+ * @brief The InfoOwner class for collect information about owner.
+ */
+class CopyrighFixer_EXPORT InfoOwner{
```

Comment:
Hmm what you think about this changes:
```suggestion
class CopyrighFixer_EXPORT Owner{
```

I think that The Owner name is more understandable.